packages/shared/src/variants/baduk.ts

```diff
@@ -170,12 +188,15 @@ export class Baduk extends AbstractGame<BadukConfig, BadukState> {
 
     this.score_board = board;
 
-    const black_points: number = board.reduce(
-      count_color<Color>(Color.BLACK),
-      0,
+    let black_points = 0;
+    board.forEach(
+      (intersection) => (black_points += intersection === Color.BLACK ? 1 : 0),
+    );
+
+    let white_points = 0;
+    board.forEach(
+      (intersection) => (white_points += intersection === Color.WHITE ? 1 : 0),
     );
```

Comment:
Alternatively, we could also have `Fillable` return a one-dimensional array and use `reduce` on that.

Reply:
I'm fine with any of these options, so I'd like to leave the decision to you.

We can also undo the changes here and continue using this type instead of fillable:

`export declare type BadukBoard<TColor> = Grid<TColor> | GraphWrapper<TColor>;`

Reply:
Oh sure if that allows us to use `reduce` again, typing as `Grid | GraphWrapper` would be nice!

In the end I don't have a strong preference - I do want to keep `reduce` so we can re-use the reducer and keep variables like `white_points` `const`, but otherwise I don't feel strongly about how we do it.

Reply:
done!